.ci_support/win_c_compilervs2015vc14.yaml

```diff
@@ -3,7 +3,7 @@ c_compiler:
 curl:
 - '7.59'
 fortran_compiler:
-- gfortran
+- flang
```

Comment:
Interesting. 🤔

cc @isuruf

Reply:
` - {{ compiler("fortran") }}  # [unix]` can now be replaced with `- {{ compiler("fortran") }}`